packages/shared/src/components/search/SearchHistoryButton.tsx

```diff
@@ -91,6 +101,9 @@ export function SearchHistoryButton(): ReactElement {
         id={contextMenuId}
         options={options}
         onHidden={() => setIsMenuOpen(false)}
+        onShown={() => {
+          trackEvent({ event_name: AnalyticsEvent.OpenSearchHistoryTooltip });
+        }}
```

Comment:
This is not validated or requested by analytics team, we can't make them up for them. 🙏